pkgs/by-name/he/helix/package.nix

```diff
@@ -1,67 +1,91 @@
 {
-  fetchzip,
+  fetchFromGitHub,
   lib,
   rustPlatform,
   git,
   installShellFiles,
   versionCheckHook,
-  nix-update-script,
+  callPackage,
+  runCommand,
 }:
 
-rustPlatform.buildRustPackage (final: {
-  pname = "helix";
-  version = "25.07.1";
+rustPlatform.buildRustPackage (
+  final:
+  let
+    tree-sitter-grammars = callPackage ./grammars.nix { };
 
-  # This release tarball includes source code for the tree-sitter grammars,
-  # which is not ordinarily part of the repository.
-  src = fetchzip {
-    url = "https://github.com/helix-editor/helix/releases/download/${final.version}/helix-${final.version}-source.tar.xz";
-    hash = "sha256-Pj/lfcQXRWqBOTTWt6+Gk61F9F1UmeCYr+26hGdG974=";
-    stripRoot = false;
-  };
+    runtimeDir = runCommand "helix-runtime" { } (
```

Comment:
This could probably be done with something like `buildEnv` rather than building the links manually.

Reply:
Apparently also `buildEnv` uses links instead of copying.

Reply:
I ended up by using `linkFarm`, I hope it's fine for you :)